db.go

```diff
@@ -40,6 +40,9 @@ const (
 // default page size for db is set to the OS page size.
 var defaultPageSize = os.Getpagesize()
 
+// When enabled, the database will perform assert function to check the slow-path code
+var assertVerify = os.Getenv("BBOLT_VERIFY") == "true"
```

Comment:
Besides env var would it be possible to configure this in another way?

Reply:
maybe we can add a parameter in Options and init the 'var assertVerify' when user call the Open method